selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsToolbar.java

```diff
@@ -206,11 +206,7 @@ public void clickOnPreviewCommandBtnAndSelectUrl(String urlCommand) {
   }
 
   private void waitListIsRenderedAndClickOnItem(String nameOfCommand, Actions action) {
-    testWebElementRenderChecker.waitElementIsRendered(
-        String.format(
-            "//div[text()='%s']/parent::div/parent::div/parent::div[@class='popupContent']",
-            nameOfCommand));
-
+    testWebElementRenderChecker.waitElementIsRendered("//div[@id='commandsPopup']");
```

Comment:
What about replacing the method `waitElementIsRendered(String webElementXpath)` with more universal one 
`waitElementIsRendered(By locator)`? It allows to use **id** itself:
```
testWebElementRenderChecker.waitElementIsRendered(By.id("commandsPopup"))
```

Reply:
This method only works with "WebElement" or "String" which provide element`s "xpath",  and you suggest passing the "By".
But I agree with you that it looks better.
So maybe I should create a new override for method "waitElementIsRendered" with "By" in argument.

Reply:
one method **waitElementIsRendered(By locator)** is enough. For xpath you can use 
`testWebElementRenderChecker.waitElementIsRendered(By.xpath(xpathString))`